docs/keepassxc-browser-manual.md

```diff
@@ -0,0 +1,333 @@
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser remember multiple credentials" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_remember_multiple_credentials.png">
+</div>
+
+New entries are stored in a separate group *KeePassXC-Browser Passwords* in the database.
+
+In *Settings -> General* the following settings are related to this feature:
+- *Blink Time* (The maximum time icon blinks after detecting a credential change)
+- *Redirect Offset* (The time icon blinks after a page redirect)
+- *Redirect Allowance* (How many redirects are allowed before deactivating the blinking icon) 
+- *Show a notification when new credentials can be saved to the database.*
+- *Save domain only.* (When saving new credentials save only the domain instead of full URL)
+
+Please notice that detecting the form changes might not work with AJAX requests.
+
+### 1.6. Choose credentials for HTTP Basic Auth requests
+
+KeePassXC-Browser can try automatically fill HTTP Basic Auth dialogs. If only one credentials are available, the dialog will be filled automatically.
+If the login attempt fails, the default login dialog will be displayed.
+
+When multiple credentials are available, the KeePassXC-Browser popup icon can be used to select the correct one. This can be reverted by clicking
+*Dismiss and show the default authentication dialog* from the popup.
+
+This feature is activated by default and can be disabled from the settings.
+
+### 1.7. Fill additional values using String Fields
+
+It is possible to fill additional information by defining String Fields to a page.
+
+1. Use *Choose custom credential fields for this page* from the popup. Username and password fields can be skipped if not needed.
+2. Now additional fields called String Fields can be choosed. Dropdown elements can be choosed also. Remember the order they are selected.
+3. After choosing the String Fields, attributes must be added to your database entry starting with `KPH: `. The attributes will be filled in that order.
+
+An example using the page [https://meine.deutsche-bank.de/trxm/db/](https://meine.deutsche-bank.de/trxm/db/):
+- *Choose custom credential fields for this page* and select Branch, Account and Sub-account as String Fields after selecting or skipping username and password field.
+- Go to your entry, Advanced and add the following attributes (in this order):
+1. `KPH: Account`
+2. `KPH: Branch`
+3. `KPH: Sub-account`
+
+Then add the values to those attributes.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser configuring String Fields" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_configuring_string_fields.png">
+</div>
+
+### 1.8. Fill TOTP
+
+KeePassXC-Browser can return Time-based One-time Passwords from the database entry. With KeePassXC 2.3.3. and newer no additional attributes are needed for the fill.
```

Comment:
It looks like the capitalization for TOTP should be "**T**ime-based **O**ne-**T**ime **P**asswords" (i.e. capitalize the "-time" bit)

Reply:
`2.3.3.` shouldn't have a period after it, I think? "With KeePassXC 2.3.3 and newer"